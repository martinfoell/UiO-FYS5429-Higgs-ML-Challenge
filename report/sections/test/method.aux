\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\citation{Murphy2012,Bishop2006,Hastie2009}
\citation{Bishop2006,Goodfellow2016,Murphy2012,Hastie2009}
\bibdata{methodNotes}
\bibstyle{apsrmp4-1}
\citation{REVTEX41Control}
\citation{apsrev41Control}
\@writefile{toc}{\contentsline {section}{\numberline {I}Artificial Intelligence and Machine Learning for nuclear physics in broad strokes}{1}{section*.1}\protected@file@percent }
\@writefile{lot}{\contentsline {table}{\numberline {I}{\ignorespaces Table of AI/ML with indication on the main type of learning (S: supervised, U: unsupervised, Semi-S: semi-supervised). }}{2}{table.1}\protected@file@percent }
\newlabel{tab:acronyms}{{I}{2}{Table of AI/ML with indication on the main type of learning (S: supervised, U: unsupervised, Semi-S: semi-supervised)}{table.1}{}}
\newlabel{LastPage}{{}{2}{}{}{}}
\gdef \@abspage@last{2}
