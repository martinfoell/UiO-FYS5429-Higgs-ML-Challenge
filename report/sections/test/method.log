This is pdfTeX, Version 3.141592653-2.6-1.40.22 (TeX Live 2022/dev/Debian) (preloaded format=pdflatex 2024.3.17)  18 MAY 2024 12:39
entering extended mode
 restricted \write18 enabled.
 file:line:error style messages enabled.
 %&-line parsing enabled.
**method.tex
(./method.tex
LaTeX2e <2021-11-15> patch level 1
L3 programming layer <2022-01-21>
(/usr/share/texlive/texmf-dist/tex/latex/subfiles/subfiles.cls
Document Class: subfiles 2020/11/14 v2.2 Multi-file projects (class)
Preamble taken from file `../../main/main.tex'
(/usr/share/texlive/texmf-dist/tex/latex/import/import.sty
Package: import 2020/04/01  v 6.2
)
(../../main/main.tex
(/usr/share/texlive/texmf-dist/tex/latex/revtex4-1/revtex4-1.cls
Document Class: revtex4-1 2020/09/30 4.1s (http://publish.aps.org/revtex4/ for 
documentation)
 Copyright (c) 2009 The American Physical Society.
 mailto:revtex@aps.org
 Licensed under the LPPL:
http://www.ctan.org/tex-archive/macros/latex/base/lppl.txt
 Arthur Ogawa <arthur_ogawa at sbcglobal dot net>
 Based on work by David Carlisle <david at dcarlisle.demon.co.uk>.
ltxutil[2020/09/30 4.1s utilities package (portions licensed from W. E. Baxter 
web at superscript.com)]
LaTeX Info: Redefining \@centercr on input line 346.
Class revtex4-1 Info: Repairing hyperref-unfriendly LaTeX definition of \mpfoot
note on input line 471.
Class revtex4-1 Info: Repairing broken LateX \@sect on input line 793.
\intertabularlinepenalty=\count185
\@tbpen=\count186
\@arstrutbox@hline=\box50
Class revtex4-1 Info: Repairing broken LaTeX \@xbitor on input line 1824.
Class revtex4-1 Info: Making \typeout \long on input line 2089.
ltxfront[2020/09/30 4.1s frontmatter package (AO,DPC)]
\c@affil=\count187
\c@collab=\count188
\absbox=\box51
ltxgrid[2020/09/30 4.1s page grid package (portions licensed from W. E. Baxter 
web at superscript.com)]
\c@linecount=\count189
\output@latex=\toks16
\box@size@1=\box52
\box@size@2=\box53
\@protection@box=\box54
\@topmark@saved=\toks17
\footins@saved=\box55
\footins@recovered=\box56
\column@recovered=\box57
\pagesofar=\box58
\footsofar=\box59
\pagegrid@col=\count190
Class revtex4-1 Info: Overriding \@vspace, \@vspacer, \@no@pgbk, \@newline, and
 \\  on input line 5655.
\c@part=\count191
\c@section=\count192
\c@subsection=\count193
\c@subsubsection=\count194
\c@paragraph=\count195
\c@subparagraph=\count196
\abovecaptionskip=\skip47
\belowcaptionskip=\skip48
\c@figure=\count197
\fbox@1=\box60
\c@table=\count198
\fbox@2=\box61
\c@video=\count199
\fbox@4=\box62
\@bibdataout=\write3
\widetext@top=\box63
\widetext@bot=\box64
Class revtex4-1 Info: Selecting society aps on input line 7616.
(/usr/share/texlive/texmf-dist/tex/latex/revtex4-1/aps4-1.rtx
File: aps4-1 2020/09/30 4.1s (http://publish.aps.org/revtex4/ for documentation
)
Class revtex4-1 Info: RevTeX society APS selected on input line 31.
)
Class revtex4-1 Info: Selecting journal rmp on input line 7618.

(/usr/share/texlive/texmf-dist/tex/latex/revtex4-1/apsrmp4-1.rtx
File: apsrmp4-1 2020/09/30 4.1s (http://publish.aps.org/revtex4/ for documentat
ion)
Class revtex4-1 Info: APS journal RMP selected on input line 35.
)
(/usr/share/texlive/texmf-dist/tex/latex/revtex4-1/aps10pt4-1.rtx
File: aps10pt4-1 2020/09/30 4.1s (http://publish.aps.org/revtex4/ for documenta
tion)
Class revtex4-1 Info: RevTeX pointsize 10pt selected on input line 30.
)
(/usr/share/texlive/texmf-dist/tex/latex/textcase/textcase.sty
Package: textcase 2019/09/14 v1.00 Text only upper/lower case changing (DPC)
)
(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amssymb.sty
Package: amssymb 2013/01/14 v3.01 AMS font symbols

(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amsfonts.sty
Package: amsfonts 2013/01/14 v3.01 Basic AMSFonts support
\@emptytoks=\toks18
\symAMSa=\mathgroup4
\symAMSb=\mathgroup5
LaTeX Font Info:    Redeclaring math symbol \hbar on input line 98.
LaTeX Font Info:    Overwriting math alphabet `\mathfrak' in version `bold'
(Font)                  U/euf/m/n --> U/euf/b/n on input line 106.
))
(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsmath.sty
Package: amsmath 2021/10/15 v2.17l AMS math features
\@mathmargin=\skip49

For additional information on amsmath, use the `?' option.
(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amstext.sty
Package: amstext 2021/08/26 v2.01 AMS text

(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsgen.sty
File: amsgen.sty 1999/11/30 v2.0 generic functions
\@emptytoks=\toks19
\ex@=\dimen138
)
LaTeX Info: Redefining \text on input line 28.
)
(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsbsy.sty
Package: amsbsy 1999/11/29 v1.2d Bold Symbols
\pmbraise@=\dimen139
)
(/usr/share/texlive/texmf-dist/tex/latex/amsmath/amsopn.sty
Package: amsopn 2021/08/26 v2.02 operator names
)
\inf@bad=\count266
LaTeX Info: Redefining \frac on input line 234.
\uproot@=\count267
\leftroot@=\count268
LaTeX Info: Redefining \overline on input line 399.
\classnum@=\count269
\DOTSCASE@=\count270
LaTeX Info: Redefining \ldots on input line 496.
LaTeX Info: Redefining \dots on input line 499.
LaTeX Info: Redefining \cdots on input line 620.
\Mathstrutbox@=\box65
\strutbox@=\box66
\big@size=\dimen140
LaTeX Font Info:    Redeclaring font encoding OML on input line 743.
LaTeX Font Info:    Redeclaring font encoding OMS on input line 744.
\macc@depth=\count271
\c@MaxMatrixCols=\count272
\dotsspace@=\muskip16
\c@parentequation=\count273
\dspbrk@lvl=\count274
\tag@help=\toks20
\row@=\count275
\column@=\count276
\maxfields@=\count277
\andhelp@=\toks21
\eqnshift@=\dimen141
\alignsep@=\dimen142
\tagshift@=\dimen143
\tagwidth@=\dimen144
\totwidth@=\dimen145
\lineht@=\dimen146
\@envbody=\toks22
\multlinegap=\skip50
\multlinetaggap=\skip51
\mathdisplay@stack=\toks23
LaTeX Info: Redefining \[ on input line 2938.
LaTeX Info: Redefining \] on input line 2939.
)
(/usr/share/texlive/texmf-dist/tex/latex/url/url.sty
\Urlmuskip=\muskip17
Package: url 2013/09/16  ver 3.4  Verb mode for urls, etc.
)
(/usr/share/texlive/texmf-dist/tex/latex/natbib/natbib.sty
Package: natbib 2010/09/13 8.31b (PWD, AO)
\bibhang=\skip52
\bibsep=\skip53
LaTeX Info: Redefining \cite on input line 694.
\c@NAT@ctr=\count278
)
Class revtex4-1 Info: Repairing natbib's \BibitemShut on input line 7623.

(/usr/share/texlive/texmf-dist/tex/latex/revtex4-1/revsymb4-1.sty
Package: revsymb4-1 2020/09/30 4.1s (http://publish.aps.org/revtex4/ for docume
ntation)
LaTeX Info: Redefining \frak on input line 95.
LaTeX Info: Redefining \Bbb on input line 111.
LaTeX Info: Redefining \REV@mathfrak on input line 119.
))
(/usr/share/texlive/texmf-dist/tex/latex/tools/bm.sty
Package: bm 2021/04/25 v1.2e Bold Symbol Support (DPC/FMi)
\symboldoperators=\mathgroup6
\symboldletters=\mathgroup7
\symboldsymbols=\mathgroup8
Package bm Info: No bold for \OMX/cmex/m/n, using \pmb.
Package bm Info: No bold for \U/msa/m/n, using \pmb.
Package bm Info: No bold for \U/msb/m/n, using \pmb.
LaTeX Font Info:    Redeclaring math alphabet \mathbf on input line 149.
LaTeX Info: Redefining \bm on input line 218.
)
(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphicx.sty
Package: graphicx 2021/09/16 v1.2d Enhanced LaTeX Graphics (DPC,SPQR)

(/usr/share/texlive/texmf-dist/tex/latex/graphics/keyval.sty
Package: keyval 2014/10/28 v1.15 key=value parser (DPC)
\KV@toks@=\toks24
)
(/usr/share/texlive/texmf-dist/tex/latex/graphics/graphics.sty
Package: graphics 2021/03/04 v1.4d Standard LaTeX Graphics (DPC,SPQR)

(/usr/share/texlive/texmf-dist/tex/latex/graphics/trig.sty
Package: trig 2021/08/11 v1.11 sin cos tan (DPC)
)
(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/graphics.cfg
File: graphics.cfg 2016/06/04 v1.11 sample graphics configuration
)
Package graphics Info: Driver file: pdftex.def on input line 107.

(/usr/share/texlive/texmf-dist/tex/latex/graphics-def/pdftex.def
File: pdftex.def 2020/10/05 v1.2a Graphics/color driver for pdftex
))
\Gin@req@height=\dimen147
\Gin@req@width=\dimen148
)
(/usr/share/texlive/texmf-dist/tex/latex/epstopdf-pkg/epstopdf.sty
Package: epstopdf 2020-01-24 v2.11 Conversion with epstopdf on the fly (HO)

(/usr/share/texlive/texmf-dist/tex/generic/infwarerr/infwarerr.sty
Package: infwarerr 2019/12/03 v1.5 Providing info/warning/error messages (HO)
)
(/usr/share/texlive/texmf-dist/tex/latex/grfext/grfext.sty
Package: grfext 2019/12/03 v1.3 Manage graphics extensions (HO)

(/usr/share/texlive/texmf-dist/tex/generic/kvdefinekeys/kvdefinekeys.sty
Package: kvdefinekeys 2019-12-19 v1.6 Define keys (HO)
))
(/usr/share/texlive/texmf-dist/tex/latex/kvoptions/kvoptions.sty
Package: kvoptions 2020-10-07 v3.14 Key value format for package options (HO)

(/usr/share/texlive/texmf-dist/tex/generic/ltxcmds/ltxcmds.sty
Package: ltxcmds 2020-05-10 v1.25 LaTeX kernel commands for general use (HO)
)
(/usr/share/texlive/texmf-dist/tex/generic/kvsetkeys/kvsetkeys.sty
Package: kvsetkeys 2019/12/15 v1.18 Key value parser (HO)
))
(/usr/share/texlive/texmf-dist/tex/generic/pdftexcmds/pdftexcmds.sty
Package: pdftexcmds 2020-06-27 v0.33 Utility functions of pdfTeX for LuaTeX (HO
)

(/usr/share/texlive/texmf-dist/tex/generic/iftex/iftex.sty
Package: iftex 2020/03/06 v1.0d TeX engine tests
)
Package pdftexcmds Info: \pdf@primitive is available.
Package pdftexcmds Info: \pdf@ifprimitive is available.
Package pdftexcmds Info: \pdfdraftmode found.
)
(/usr/share/texlive/texmf-dist/tex/latex/epstopdf-pkg/epstopdf-base.sty
Package: epstopdf-base 2020-01-24 v2.11 Base part for package epstopdf
Package epstopdf-base Info: Redefining graphics rule for `.eps' on input line 4
85.
Package grfext Info: Graphics extension search list:
(grfext)             [.pdf,.png,.jpg,.mps,.jpeg,.jbig2,.jb2,.PDF,.PNG,.JPG,.JPE
G,.JBIG2,.JB2,.eps]
(grfext)             \AppendGraphicsExtensions on input line 504.

(/usr/share/texlive/texmf-dist/tex/latex/latexconfig/epstopdf-sys.cfg
File: epstopdf-sys.cfg 2010/07/13 v1.3 Configuration of (r)epstopdf for TeX Liv
e
)))
(/usr/share/texlive/texmf-dist/tex/latex/wrapfig/wrapfig.sty
\wrapoverhang=\dimen149
\WF@size=\dimen150
\c@WF@wrappedlines=\count279
\WF@box=\box67
\WF@everypar=\toks25
Package: wrapfig 2003/01/31  v 3.6
)
(/usr/share/texlive/texmf-dist/tex/latex/tools/array.sty
Package: array 2021/10/04 v2.5f Tabular extension package (FMi)
\col@sep=\dimen151
\ar@mcellbox=\box68
\extrarowheight=\dimen152
\NC@list=\toks26
\extratabsurround=\skip54
\backup@length=\skip55
\ar@cellbox=\box69
)
(/usr/share/texlive/texmf-dist/tex/latex/listings/listings.sty
\lst@mode=\count280
\lst@gtempboxa=\box70
\lst@token=\toks27
\lst@length=\count281
\lst@currlwidth=\dimen153
\lst@column=\count282
\lst@pos=\count283
\lst@lostspace=\dimen154
\lst@width=\dimen155
\lst@newlines=\count284
\lst@lineno=\count285
\lst@maxwidth=\dimen156

(/usr/share/texlive/texmf-dist/tex/latex/listings/lstmisc.sty
File: lstmisc.sty 2020/03/24 1.8d (Carsten Heinz)
\c@lstnumber=\count286
\lst@skipnumbers=\count287
\lst@framebox=\box71
)
(/usr/share/texlive/texmf-dist/tex/latex/listings/listings.cfg
File: listings.cfg 2020/03/24 1.8d listings configuration
))
Package: listings 2020/03/24 1.8d (Carsten Heinz)

(/usr/share/texlive/texmf-dist/tex/latex/threeparttablex/threeparttablex.sty
Package: threeparttablex 2013/07/23 v0.3 by daleif

(/usr/share/texlive/texmf-dist/tex/latex/threeparttable/threeparttable.sty
Package: threeparttable 2003/06/13  v 3.0
\@tempboxb=\box72
)
(/usr/share/texlive/texmf-dist/tex/latex/environ/environ.sty
Package: environ 2014/05/04 v0.3 A new way to define environments

(/usr/share/texlive/texmf-dist/tex/latex/trimspaces/trimspaces.sty
Package: trimspaces 2009/09/17 v1.1 Trim spaces around a token list
))
\TPTL@width=\skip56
)
(/usr/share/texlive/texmf-dist/tex/latex/booktabs/booktabs.sty
Package: booktabs 2020/01/12 v1.61803398 Publication quality tables
\heavyrulewidth=\dimen157
\lightrulewidth=\dimen158
\cmidrulewidth=\dimen159
\belowrulesep=\dimen160
\belowbottomsep=\dimen161
\aboverulesep=\dimen162
\abovetopsep=\dimen163
\cmidrulesep=\dimen164
\cmidrulekern=\dimen165
\defaultaddspace=\dimen166
\@cmidla=\count288
\@cmidlb=\count289
\@aboverulesep=\dimen167
\@belowrulesep=\dimen168
\@thisruleclass=\count290
\@lastruleclass=\count291
\@thisrulewidth=\dimen169
)
(/usr/share/texlive/texmf-dist/tex/latex/tools/dcolumn.sty
Package: dcolumn 2014/10/28 v1.06 decimal alignment package (DPC)
)
(/usr/share/texlive/texmf-dist/tex/latex/graphics/color.sty
Package: color 2021/12/07 v1.3c Standard LaTeX Color (DPC)

(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/color.cfg
File: color.cfg 2016/01/02 v1.6 sample color configuration
)
Package color Info: Driver file: pdftex.def on input line 149.
)
(/usr/share/texlive/texmf-dist/tex/latex/textpos/textpos.sty
Package: textpos 2020/09/26 v1.10

Package: textpos 2020/09/26 1.10, absolute positioning of text on the page
Package textpos Info: choosing support for LaTeX3 on input line 61.
\TP@textbox=\box73
\TPHorizModule=\dimen170
\TPVertModule=\dimen171
\TP@margin=\dimen172
\TP@absmargin=\dimen173
Grid set 16 x 16 = 38.39343pt x 49.68562pt
\TPboxrulesize=\dimen174
\TP@tbargs=\toks28
) (/usr/share/texlive/texmf-dist/tex/latex/multirow/multirow.sty
Package: multirow 2021/03/15 v2.8 Span multiple rows of a table
\multirow@colwidth=\skip57
\multirow@cntb=\count292
\multirow@dima=\skip58
\bigstrutjot=\dimen175
)
(/usr/share/texlive/texmf-dist/tex/latex/multirow/bigdelim.sty
Package: bigdelim 2021/03/15 v2.8 Create big delimiters in tabular or array
)
(/usr/share/texlive/texmf-dist/tex/latex/float/float.sty
Package: float 2001/11/08 v1.3d Float enhancements (AL)
\c@float@type=\count293
\float@exts=\toks29
\float@box=\box74
\@float@everytoks=\toks30
\@floatcapt=\box75
)
(/usr/share/texlive/texmf-dist/tex/latex/was/upgreek.sty
Package: upgreek 2003/02/12 v2.0 (WaS)
Package upgreek Info: Using Euler Roman for upright Greek on input line 31.
\symugrf@m=\mathgroup9
LaTeX Font Info:    Overwriting symbol font `ugrf@m' in version `bold'
(Font)                  U/eur/m/n --> U/eur/b/n on input line 38.
)
(/usr/share/texlive/texmf-dist/tex/latex/base/inputenc.sty
Package: inputenc 2021/02/14 v1.3d Input encoding file
\inpenc@prehook=\toks31
\inpenc@posthook=\toks32
)
(/usr/share/texlive/texmf-dist/tex/latex/hyperref/hyperref.sty
Package: hyperref 2021-06-07 v7.00m Hypertext links for LaTeX

(/usr/share/texlive/texmf-dist/tex/generic/pdfescape/pdfescape.sty
Package: pdfescape 2019/12/09 v1.15 Implements pdfTeX's escape features (HO)
)
(/usr/share/texlive/texmf-dist/tex/latex/hycolor/hycolor.sty
Package: hycolor 2020-01-27 v1.10 Color options for hyperref/bookmark (HO)
)
(/usr/share/texlive/texmf-dist/tex/latex/letltxmacro/letltxmacro.sty
Package: letltxmacro 2019/12/03 v1.6 Let assignment for LaTeX macros (HO)
)
(/usr/share/texlive/texmf-dist/tex/latex/auxhook/auxhook.sty
Package: auxhook 2019-12-17 v1.6 Hooks for auxiliary files (HO)
)
\@linkdim=\dimen176
\Hy@linkcounter=\count294
\Hy@pagecounter=\count295

(/usr/share/texlive/texmf-dist/tex/latex/hyperref/pd1enc.def
File: pd1enc.def 2021-06-07 v7.00m Hyperref: PDFDocEncoding definition (HO)
Now handling font encoding PD1 ...
... no UTF-8 mapping file for font encoding PD1
)
(/usr/share/texlive/texmf-dist/tex/latex/hyperref/hyperref-langpatches.def
File: hyperref-langpatches.def 2021-06-07 v7.00m Hyperref: patches for babel la
nguages
)
(/usr/share/texlive/texmf-dist/tex/generic/intcalc/intcalc.sty
Package: intcalc 2019/12/15 v1.3 Expandable calculations with integers (HO)
)
(/usr/share/texlive/texmf-dist/tex/generic/etexcmds/etexcmds.sty
Package: etexcmds 2019/12/15 v1.7 Avoid name clashes with e-TeX commands (HO)
)
\Hy@SavedSpaceFactor=\count296

(/usr/share/texlive/texmf-dist/tex/latex/hyperref/puenc.def
File: puenc.def 2021-06-07 v7.00m Hyperref: PDF Unicode definition (HO)
Now handling font encoding PU ...
... no UTF-8 mapping file for font encoding PU
)
Package hyperref Info: Hyper figures OFF on input line 4192.
Package hyperref Info: Link nesting OFF on input line 4197.
Package hyperref Info: Hyper index ON on input line 4200.
Package hyperref Info: Plain pages OFF on input line 4207.
Package hyperref Info: Backreferencing OFF on input line 4212.
Package hyperref Info: Implicit mode ON; LaTeX internals redefined.
Package hyperref Info: Bookmarks ON on input line 4445.
LaTeX Info: Redefining \href on input line 4715.
\c@Hy@tempcnt=\count297
LaTeX Info: Redefining \url on input line 4804.
\XeTeXLinkMargin=\dimen177

(/usr/share/texlive/texmf-dist/tex/generic/bitset/bitset.sty
Package: bitset 2019/12/09 v1.3 Handle bit-vector datatype (HO)

(/usr/share/texlive/texmf-dist/tex/generic/bigintcalc/bigintcalc.sty
Package: bigintcalc 2019/12/15 v1.5 Expandable calculations on big integers (HO
)
))
\Fld@menulength=\count298
\Field@Width=\dimen178
\Fld@charsize=\dimen179
Package hyperref Info: Hyper figures OFF on input line 6076.
Package hyperref Info: Link nesting OFF on input line 6081.
Package hyperref Info: Hyper index ON on input line 6084.
Package hyperref Info: backreferencing OFF on input line 6091.
Package hyperref Info: Link coloring OFF on input line 6096.
Package hyperref Info: Link coloring with OCG OFF on input line 6101.
Package hyperref Info: PDF/A mode OFF on input line 6106.
LaTeX Info: Redefining \ref on input line 6146.
LaTeX Info: Redefining \pageref on input line 6150.

(/usr/share/texlive/texmf-dist/tex/latex/base/atbegshi-ltx.sty
Package: atbegshi-ltx 2021/01/10 v1.0c Emulation of the original atbegshi
package with kernel methods
)
\Hy@abspage=\count299
\c@Item=\count300
\c@Hfootnote=\count301
)
Package hyperref Info: Driver (autodetected): hpdftex.

(/usr/share/texlive/texmf-dist/tex/latex/hyperref/hpdftex.def
File: hpdftex.def 2021-06-07 v7.00m Hyperref driver for pdfTeX

(/usr/share/texlive/texmf-dist/tex/latex/base/atveryend-ltx.sty
Package: atveryend-ltx 2020/08/19 v1.0a Emulation of the original atveryend pac
kage
with kernel methods
)
\Fld@listcount=\count302
\c@bookmark@seq@number=\count303

(/usr/share/texlive/texmf-dist/tex/latex/rerunfilecheck/rerunfilecheck.sty
Package: rerunfilecheck 2019/12/05 v1.9 Rerun checks for auxiliary files (HO)

(/usr/share/texlive/texmf-dist/tex/generic/uniquecounter/uniquecounter.sty
Package: uniquecounter 2019/12/15 v1.4 Provide unlimited unique counter (HO)
)
Package uniquecounter Info: New unique counter `rerunfilecheck' on input line 2
86.
)
\Hy@SectionHShift=\skip59
)
Package hyperref Info: Option `breaklinks' set `true' on input line 23.
Package hyperref Info: Option `colorlinks' set `true' on input line 23.

(/usr/share/texlive/texmf-dist/tex/latex/xcolor/xcolor.sty
Package: xcolor 2021/10/31 v2.13 LaTeX color extensions (UK)

(/usr/share/texlive/texmf-dist/tex/latex/graphics-cfg/color.cfg
File: color.cfg 2016/01/02 v1.6 sample color configuration
)
Package xcolor Info: Driver file: pdftex.def on input line 227.
LaTeX Info: Redefining \color on input line 711.
Package xcolor Info: Model `cmy' substituted by `cmy0' on input line 1352.
Package xcolor Info: Model `hsb' substituted by `rgb' on input line 1356.
Package xcolor Info: Model `RGB' extended on input line 1368.
Package xcolor Info: Model `HTML' substituted by `rgb' on input line 1370.
Package xcolor Info: Model `Hsb' substituted by `hsb' on input line 1371.
Package xcolor Info: Model `tHsb' substituted by `hsb' on input line 1372.
Package xcolor Info: Model `HSB' substituted by `hsb' on input line 1373.
Package xcolor Info: Model `Gray' substituted by `gray' on input line 1374.
Package xcolor Info: Model `wave' substituted by `hsb' on input line 1375.
)
(/usr/share/texlive/texmf-dist/tex/latex/subfiles/subfiles.sty
Package: subfiles 2020/11/14 v2.2 Multi-file projects (package)
)))
Class revtex4-1 Info: Unrecognized array package. Please update this document c
lass! (Proceeding with fingers crossed.) on input line 3.


Class revtex4-1 Warning: Repair the float package.

Class revtex4-1 Info: cite was not loaded (OK!) on input line 3.
Class revtex4-1 Info: mcite was not loaded (OK!) on input line 3.
Class revtex4-1 Info: multicol was not loaded (OK!) on input line 3.
(/usr/share/texlive/texmf-dist/tex/latex/l3backend/l3backend-pdftex.def
File: l3backend-pdftex.def 2022-01-12 L3 backend support: PDF output (pdfTeX)
\l__color_backend_stack_int=\count304
\l__pdf_internal_box=\box76
)

LaTeX Warning: Unused global option(s):
    [graphicx].

(./method.aux)
\openout1 = `method.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for TS1/cmr/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for PD1/pdf/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for PU/pdf/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.

(/usr/share/texlive/texmf-dist/tex/context/base/mkii/supp-pdf.mkii
[Loading MPS to PDF converter (version 2006.09.02).]
\scratchcounter=\count305
\scratchdimen=\dimen180
\scratchbox=\box77
\nofMPsegments=\count306
\nofMParguments=\count307
\everyMPshowfont=\toks33
\MPscratchCnt=\count308
\MPscratchDim=\dimen181
\MPnumerator=\count309
\makeMPintoPDFobject=\count310
\everyMPtoPDFconversion=\toks34
)
\c@lstlisting=\count311
Package hyperref Info: Link coloring ON on input line 3.
 (/usr/share/texlive/texmf-dist/tex/latex/hyperref/nameref.sty
Package: nameref 2021-04-02 v2.47 Cross-referencing by name of section

(/usr/share/texlive/texmf-dist/tex/latex/refcount/refcount.sty
Package: refcount 2019/12/15 v3.6 Data extraction from label references (HO)
)
(/usr/share/texlive/texmf-dist/tex/generic/gettitlestring/gettitlestring.sty
Package: gettitlestring 2019/12/15 v1.6 Cleanup title references (HO)
)
\c@section@level=\count312
)
LaTeX Info: Redefining \ref on input line 3.
LaTeX Info: Redefining \pageref on input line 3.
LaTeX Info: Redefining \nameref on input line 3.

(./method.out) (./method.out)
\@outlinefile=\write4
\openout4 = `method.out'.

Class revtex4-1 Info: Taking \maketitle back from hyperref on input line 3.
\openout3 = `methodNotes.bib'.

LaTeX Font Info:    Trying to load font information for U+msa on input line 69.


(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/umsa.fd
File: umsa.fd 2013/01/14 v3.01 AMS symbols A
)
LaTeX Font Info:    Trying to load font information for U+msb on input line 69.


(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/umsb.fd
File: umsb.fd 2013/01/14 v3.01 AMS symbols B
)

Package natbib Warning: Citation `Murphy2012' on page 1 undefined on input line
 118.


Package natbib Warning: Citation `Bishop2006' on page 1 undefined on input line
 118.


Package natbib Warning: Citation `Hastie2009' on page 1 undefined on input line
 118.


Package natbib Warning: Citation `Bishop2006' on page 1 undefined on input line
 138.


Package natbib Warning: Citation `Goodfellow2016' on page 1 undefined on input 
line 138.


Package natbib Warning: Citation `Murphy2012' on page 1 undefined on input line
 138.


Package natbib Warning: Citation `Hastie2009' on page 1 undefined on input line
 138.

(./method.bbl)

Package natbib Warning: There were undefined citations.

[1{/var/lib/texmf/fonts/map/pdftex/updmap/pdftex.map}] [2] (./method.aux)
Package rerunfilecheck Info: File `method.out' has not changed.
(rerunfilecheck)             Checksum: 1C15EE0EA00EEA095469502DACFF3305;479.
 ) 
Here is how much of TeX's memory you used:
 14181 strings out of 480129
 216842 string characters out of 5893409
 543581 words of memory out of 5000000
 31896 multiletter control sequences out of 15000+600000
 477525 words of font info for 58 fonts, out of 8000000 for 9000
 81 hyphenation exceptions out of 8191
 89i,9n,93p,521b,339s stack positions out of 5000i,500n,10000p,200000b,80000s
</usr/share/texlive/texmf-dist/fonts/type1/public/amsfonts/cm/cmbx10.pfb></us
r/share/texlive/texmf-dist/fonts/type1/public/amsfonts/cm/cmmi9.pfb></usr/share
/texlive/texmf-dist/fonts/type1/public/amsfonts/cm/cmr10.pfb></usr/share/texliv
e/texmf-dist/fonts/type1/public/amsfonts/cm/cmr9.pfb></usr/share/texlive/texmf-
dist/fonts/type1/public/amsfonts/cm/cmssbx10.pfb>
Output written on method.pdf (2 pages, 79217 bytes).
PDF statistics:
 49 PDF objects out of 1000 (max. 8388607)
 34 compressed objects within 1 object stream
 5 named destinations out of 1000 (max. 500000)
 9 words of extra memory for PDF output out of 10000 (max. 10000000)

