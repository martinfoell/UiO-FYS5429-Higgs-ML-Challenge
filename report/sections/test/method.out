\BOOKMARK [1][-]{section*.1}{\376\377\000A\000r\000t\000i\000f\000i\000c\000i\000a\000l\000\040\000I\000n\000t\000e\000l\000l\000i\000g\000e\000n\000c\000e\000\040\000a\000n\000d\000\040\000M\000a\000c\000h\000i\000n\000e\000\040\000L\000e\000a\000r\000n\000i\000n\000g\000\040\000f\000o\000r\000\040\000n\000u\000c\000l\000e\000a\000r\000\040\000p\000h\000y\000s\000i\000c\000s\000\040\000i\000n\000\040\000b\000r\000o\000a\000d\000\040\000s\000t\000r\000o\000k\000e\000s}{}% 1
